client/src/reducers/people/peopleReducer.js

```diff
@@ -1,9 +1,11 @@
 import ActionTypes from '../../actions/actionTypes';
+import _ from 'lodash';
 
 const peopleReducer = (state = [], action) => {
   switch (action.type) {
     case ActionTypes.RECEIVE_PEOPLE:
-      return [...action.response.results];
```

Comment:
So now we're just adding onto the array whenever there is a new set of people. This is a problem for a few reasons, and it's where we will start digging into combined reducers and using better selectors.